packages/packages/api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/api",
-  "version": "4.0.2",
+  "version": "4.0.3-alpha.1",
```

Comment:
The change on the doc query schema is a breaking change. Even if we keep the old one around, I think we should at least increase on `minor`.

Reply:
Gotcha i updated the release plan to make sure we upgrade to minor version

Reply:
I see we're planning to update `minor`. If we decide to not make it backwards compatible, we should probably increase major.